udata/frontend/markdown.py

```diff
@@ -66,7 +66,8 @@ def mdstrip(value, length=None):
             return ''
         if EXCERPT_TOKEN in value:
             value = value.split(EXCERPT_TOKEN, 1)[0]
-        if length > 0:
-            value = do_truncate(value, length, end='…')
         rendered = md(value)
-        return do_striptags(rendered)
+        text = do_striptags(rendered)
+        if length > 0:
+            text = do_truncate(text, length, end='…')
```

Comment:
Maybe add the `end` parameter to the `mdstrip` method? There is a use-case we talked about last week but I can't remember where…